posthog/internal_metrics/team.py

```diff
@@ -353,7 +355,6 @@ def get_internal_metrics_team_id() -> Optional[int]:
         return None
 
 
-@lru_cache(maxsize=1)
```

Comment:
Alternatively we could consider clearing the cache after team_id is set (sth similar to https://stackoverflow.com/questions/45282631/clearing-lru-cache-of-certain-methods-when-an-attribute-of-the-class-is-updated/45283290#45283290)

Reply:
The issue is more around there being multiple web servers running in parallel, each caching its own value until race.

A time-based cache would work, but given this is only requested rarely (compared to team_id) it's fine to just remove it.